Manage HTML entities in timed text tracks (#552)
openfun/marsha: docs/env.md

- Default: `"eu-west-1"`
- Choices: Any valid AWS region name.

#### DJANGO_AWS_BASE_NAME

> All the AWS services we used have their name prefixed. The prefix is the terraform worspace name.

Reviewer: ```suggestion
All the AWS services we use have their name prefixed by the terraform workspace name.
```

> This setting should contain the terraform workspace name defined in marsha's aws project.

Reviewer: ```suggestion
This setting should contain the terraform workspace name defined in marsha's AWS project.
```
